### PR TITLE
Update TAXII2 Server for EDL

### PR DESCRIPTION
**Pull request by ShirleyDenkberg**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 1]
```diff
@@ -20,10 +20,16 @@ You can add a collection description as is done in *collection1_name*, or enter
...
 ## How to Access the TAXII2 Server
 
-Use one of the following options:
+(For Cortex XSOAR 6.x) Use one of the following options:
+
 - **https://*demisto_address*/instance/execute/*instance_name*/{taxii2_api_endpoint}/**
 - **http://*demisto_address*:*listen_port*/{taxii2_api_endpoint}/**
 
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
+- **https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/{taxii2_api_endpoint}/**
```
Comment (Ni-Knight) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:29]:
```suggestion
- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 2]
```diff
@@ -20,10 +20,16 @@ You can add a collection description as is done in *collection1_name*, or enter
 
 ## How to Access the TAXII2 Server
 
-Use one of the following options:
+(For Cortex XSOAR 6.x) Use one of the following options:
+
 - **https://*demisto_address*/instance/execute/*instance_name*/{taxii2_api_endpoint}/**
```
Comment (Ni-Knight) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:25]:
```suggestion
- `https://<demisto_address>/instance/execute/<instance_name>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 3]
```diff
@@ -20,10 +20,16 @@ You can add a collection description as is done in *collection1_name*, or enter
...
 - **http://*demisto_address*:*listen_port*/{taxii2_api_endpoint}/**
```
Comment (Ni-Knight) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:26]:
```suggestion
- `http://<demisto_address>:<listen_port>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 4]
```diff
@@ -20,10 +20,16 @@ You can add a collection description as is done in *collection1_name*, or enter
...
 
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
+- **https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/{taxii2_api_endpoint}/**
+- **http://demisto_address:listen_port/{taxii2_api_endpoint}/**
```
Comment (Ni-Knight) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:30]:
```suggestion
- `http://<demisto_address>:<listen_port>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 5]
```diff
@@ -152,10 +158,15 @@ TIM fields (system generated and custom). An example of these two related object
...
-  2. You can use the `!taxii-server-list-collections` command in order to get a list of your server's collections and their ids. You can also do it manually by running `curl https://<xsoar-server>/instance/execute/<instance_name>/threatintel/collections/ | jq .` to get a list  of the collections available and on your TAXII server. From the list, copy the correct ID of the collection you want to ingest. 
- 
+  2. You can use the `!taxii-server-list-collections` command in order to get a list of your server's collections and their IDs. You can also do it manually by running `curl https://<xsoar-server>/instance/execute/<instance_name>/threatintel/collections/ | jq .` to get a list  of the collections available and on your TAXII server. From the list, copy the correct ID of the collection you want to ingest. 
+
+**For Cortex XSOAR 8 and Cortex XSIAM**
+  1. All your server info can be found by running `!taxii-server-info`, the default API root for you server will usually be - https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/threatintel
```
Comment (Ni-Knight) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:166]:
```suggestion
  1. All your server info can be found by running `!taxii-server-info`, the default API root for you server will usually be - `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/threatintel/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 6]
```diff
@@ -152,10 +158,15 @@ TIM fields (system generated and custom). An example of these two related object
 
 
 #### Find the information required for the Sentinel TAXII connector
-
+**For Cortex XSOAR 6.x:**
   1. All your server info can be found by running `!taxii-server-info`, the default API root for you server will usually be - https://&lt;xsoar-server&gt;/instance/execute/&lt;instance_name&gt;/threatintel/
```
Comment (Ni-Knight) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:162]:
```suggestion
  1. All your server info can be found by running `!taxii-server-info`, the default API root for you server will usually be - `https://<xsoar-server>/instance/execute/<instance_name>/threatintel/`
```

**Review by Ni-Knight — COMMENTED** [review]
Just small changes as MD will ignore `<something>` that are not in inline mode.

**Review by ShahafBenYakir — COMMENTED** [review]
let's replace the word demisto with xsoar

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 7]
```diff
@@ -20,9 +20,15 @@ You can add a collection description as is done in *collection1_name*, or enter
 
 ## How to Access the TAXII2 Server
 
-Use one of the following options:
-- **https://*demisto_address*/instance/execute/*instance_name*/{taxii2_api_endpoint}/**
-- **http://*demisto_address*:*listen_port*/{taxii2_api_endpoint}/**
+(For Cortex XSOAR 6.x) Use one of the following options:
+
+- `https://<demisto_address>/instance/execute/<instance_name>/<taxii2_api_endpoint>/`
+- `http://<demisto_address>:<listen_port>/<taxii2_api_endpoint>/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:26]:
```suggestion
- `https://<xsoar_address>/instance/execute/<instance_name>/<taxii2_api_endpoint>/`
- `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/README.md [hunk 8]
```diff
@@ -20,9 +20,15 @@ You can add a collection description as is done in *collection1_name*, or enter
...
+
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options:
+- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
+- `http://<demisto_address>:<listen_port>/<taxii2_api_endpoint>/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/README.md:30]:
```suggestion
- `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 9]
```diff
@@ -21,15 +21,25 @@ You can add a collection description as is doen in `collection1_name`, or enter
...
-Use one of the following options:
-- **https://*demisto_address*/instance/execute/*instance_name/{taxii2_api_endpoint}/*** 
-- **http://*demisto_address*:*listen_port/{taxii2_api_endpoint}/***
+(For Cortex XSOAR 6.x) Use one of the following options:
+- **https://*demisto_address*/instance/execute/*instance_name/{taxii2_api_endpoint}/**
+- **http://*demisto_address*:*listen_port/{taxii2_api_endpoint}/**
+
+(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options: 
+- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
+- `http://<demisto_address>:<listen_port>/<taxii2_api_endpoint>/`
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:30]:
```suggestion
- **https://*xsoar_address*/instance/execute/*instance_name/{taxii2_api_endpoint}/**
- **http://*xsoar_address*:*listen_port/{taxii2_api_endpoint}/**

(For Cortex XSOAR 8 and Cortex XSIAM) Use one of the following options: 
- `https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>/<taxii2_api_endpoint>/`
- `http://<xsoar_address>:<listen_port>/<taxii2_api_endpoint>/`
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 10]
```diff
@@ -21,15 +21,25 @@ You can add a collection description as is doen in `collection1_name`, or enter
...
 
 ## Access the TAXII Service by Instance Name
 To access the TAXII service by instance name, make sure *Instance execute external* is enabled. 
 
 1. In Cortex XSOAR, go to **Settings > About > Troubleshooting**.
-2. In the **Server Configuration** section, verify that the *instance.execute.external* key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+2. (For Cortex XSOAR 6.x only) In the **Server Configuration** section, verify that the *instance.execute.external* key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*.
+3. You can now trigger the webhook URL:
+
+   - (For Cortex XSOAR 6) In a web browser, go to https://*<demisto_address>*/instance/execute/*<instance_name>* .
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:40]:
```suggestion
   - (For Cortex XSOAR 6) In a web browser, go to https://*<xsoar_address>*/instance/execute/*<instance_name>* .
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.yml [hunk 11]
```diff
@@ -23,7 +23,7 @@ configuration:
   required: true
   type: 15
   section: Connect
-- additionalinfo: Will run the TAXII2 server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances.
+- additionalinfo: Will run the TAXII2 server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for the EDL will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.yml:26]:
```suggestion
- additionalinfo: Will run the TAXII2 Server on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. (For Cortex XSOAR 8 and Cortex XSIAM) If you do not enter a Listen Port, an unused port for TAXII2 Server will automatically be generated when the instance is saved. However, if using an engine, you must enter a Listen Port.  
```

File: Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md [hunk 12]
```diff
@@ -21,15 +21,25 @@ You can add a collection description as is doen in `collection1_name`, or enter
...
+   - (For Cortex XSOAR 8 and Cortex XSIAM) In a web browser, go to https://ext-<cortex-xsiam-address>/xsoar/instance/execute/<instance-name>.
```
Comment (ShirleyDenkberg) [inline Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server_description.md:41]:
```suggestion
   -  (For Cortex XSOAR 8) go to `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
   - (For Cortex XSIAM) go to `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```

**Review by ShahafBenYakir — COMMENTED** [review]
- Under command taxii-server-info: The context path TAXIIServer.ServerInfo.description description should end with a period.

**Pull request merged**